crates/ide-db/src/syntax_helpers/node_ext.rs

```diff
@@ -457,3 +457,8 @@ pub fn parse_tt_as_comma_sep_paths(input: ast::TokenTree) -> Option<Vec<ast::Pat
         .collect();
     Some(paths)
 }
+
+pub fn get_outer_macro(string: &ast::String) -> Option<MacroCall> {
```

Comment:
```suggestion
pub fn macro_call_for_string_token(string: &ast::String) -> Option<MacroCall> {
```
This might be a bit more descriptive

Reply:
done